README.md

```diff
@@ -141,7 +141,8 @@ BaseTableInfo info = bigquery.getTable(tableId);
 if (info == null) {
   System.out.println("Creating table " + tableId);
   Field integerField = Field.of("fieldName", Field.Type.integer());
-  bigquery.create(TableInfo.of(tableId, Schema.of(integerField)));
+  Schema schema = Schema.of(integerField);
```

Comment:
A couple lines above this line there's a reference to "BaseTableInfo" that should be changed.